lib/orbf/rules_engine/fetch_data/fetch_data_analytics.rb

```diff
@@ -0,0 +1,71 @@
+module Orbf
+  module RulesEngine
+    class FetchDataAnalytics
+      def initialize(dhis2_connection, package_arguments)
+        @package_arguments = package_arguments
+        @dhis2_connection = dhis2_connection
+      end
+
+      # TODO: SMS ask piet ;)
+      #    - handle too long urls ?
+      def call
+        return [] if analytics_activity_states.none?
+
+        analytics_values = dhis2_connection.analytics.list(
```

Comment:
`dhis_connection.analaytics` just returns an array? Not a hash?

Reply:
hash with "rows"... will rename to analytics_response